services/director-v2/src/simcore_service_director_v2/models/schemas/dynamic_services/scheduler.py

```diff
@@ -295,6 +295,11 @@ class SchedulerData(CommonServiceDetails, DynamicSidecarServiceLabels):
         ..., description="Name of the current dynamic-sidecar being observed"
     )
 
+    observation_id: UUID = Field(
```

Comment:
THOUGHT
IMO "observation" is not very descriptive. Even in the context of the scheduler. In addition, you introduce yet another very generic term which is "resources". In what sense? 

I think the problem is that lack a bit of "narrative" in our designs. We do not follow a clear "story" where we name things and instead have "partial" stories where we either rename or use a generic name for the terms we need.

I wonder if we could write some narrative in the header of the scheduler. For instance

> On of the main responsibilities of the director-v2 service is to **schedule** dynamic services (dy-service). A scheduler instance is created and runs during app's lifetime for that purpose.
...

I will try to do that myself. It is a good exercise for me and you can later review it. 

When a user issues a new dy-service